tests/test_resource_realtimeapi.py

```diff
@@ -25,7 +25,7 @@ def _get_class_path(clsname):
         return '{}.{}.{}'.format(__package__, module_name, clsname)
 
     def test_realtimeapi_with_default_settings(self):
-        expected_entities = {'crawl.json': CrawlResource}
+        expected_entities = {b'crawl.json': CrawlResource}
```

Comment:
actually this is one thing where something may break for users and I'm little worried about this part, if route for resource is not bytes Twisted doesn't recognize resource path and returns 404 Not Found if you ask for crawl.json in Python 3. I made it bytes here, but actually maybe better approach would be to add encode() here https://github.com/scrapinghub/scrapyrt/blob/master/scrapyrt/resources.py#L100 this way if user has route that is not bytes but str it would still work